source/transactions/tests/README.rst

```diff
@@ -86,7 +86,15 @@ Each YAML file has the following keys:
   - ``skipReason``: Optional, string describing why this test should be
     skipped.
 
-  - ``clientOptions``: Optional, parameters to pass to MongoClient().
+  - ``topology``: Optional. An array of server topologies against which to run
+    the test. Valid topologies are "replicaset" and "sharded". The default is
+    ["replicaset", "sharded"].
+
+  - ``clientOptions``: Optional, parameters to pass to MongoClient():
+
+    - ``useMultipleMongoses``: Optional, boolean. If true and this test is
```

Comment:
This doesn't seem appropriate as a client option.   Consider making it top-level.

Reply:
I'm also not clear what the point is.  Since at least some of the sharded-only tests require this to be true, to pass any test run a driver will need to have multiple mongos available.  So can we just say that the sharded cluster tests require multiple mongos servers in the URI?

Reply:
OK, still doesn't seem like a client option though.  I think of those as things like "retryReads=true"

Reply:
I'm going to resolve this conversation since there is a similar thread below. I'm fine with whatever you decide here.